test/minikube/minikube_rolling_upgrade_test.go

```diff
@@ -31,6 +32,114 @@ func verifyAllProcessesRunning(t *testing.T, namespaceName string, adminPod stri
 	}, 30*time.Second)
 }
 
+func TestAdminReadinessProbe(t *testing.T) {
+	// TODO: remove this whenever the image tested in nuodb-helm-charts CI
+	// supports 'nuocmd check server' (singular)
+	if os.Getenv("NUODB_DEV") != "true" {
+		t.Skip("'nuocmd check server' is not supported in released versions")
+	}
+
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	// create a two-server domain and induce a failure that makes it
+	// impossible to elect a leader, causing 'nuocmd check server
+	// --check-converged' to fail
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &helm.Options{
+		SetValues: map[string]string{
+			"admin.replicas": "2",
+		},
+	}, 2, "")
+	adminStatefulSet := helmChartReleaseName + "-nuodb-cluster0"
+	admin0 := adminStatefulSet + "-0"
+	admin1 := adminStatefulSet + "-1"
+
+	// make sure readinessprobe succeeds on both Admin processes
+	options := k8s.NewKubectlOptions("", "", namespaceName)
+	output, err := k8s.RunKubectlAndGetOutputE(t, options, "exec", admin0, "--", "readinessprobe")
```

Comment:
It's it better to check that the pod is READY in addition/instead of running the prob directly?

Reply:
I think `testlib.StartAdmin()` blocks until the Admin Pods are READY, but I can check that explicitly here.

Reply:
you dont need anything between lines 58 and here. All of that is done as part of `StartAdmin`